nailgun/entities.py

```diff
@@ -1897,6 +1897,20 @@ def read(self, entity=None, attrs=None, ignore=None, params=None):
         return super(ContentViewFilterRule, self).read(
             entity, attrs, ignore, params)
 
+    def create_payload(self):
+        """Reset ``errata_id`` from DB ID to ``errata_id``."""
+        payload = super(ContentViewFilterRule, self).create_payload()
+        if 'errata_id' in payload:
+            payload['errata_id'] = self.errata.errata_id
```

Comment:
`errata` may be passed not via `.read()` or `.search()`, but also by simple `entities.Errata(id=123)`.
This way it won't have `errata_id` attr, so extra check is needed:
```python
if not hasattr(self.errata, 'errata_id'):
    self.errata = self.errata.read()
payload['errata_id'] = self.errata.errata_id
```